pkg/replication/replication.go

```diff
@@ -278,6 +288,20 @@ func (r replication) handleRenewalErrors(session kp.Session, renewalErrCh chan e
 
 // note: logging should be delegated somehow
 func (r replication) updateOne(node types.NodeName, done chan<- types.NodeName, quitCh <-chan struct{}, aggregateHealth *podHealth) {
+	defer func() {
```

Comment:
why this change? you don't want to pass a node to `done` unless you've actually updated it right? there are plenty of scenarios (e.g. a quit) where that wouldn't have happened